.vscode/extensions.json

```diff
@@ -5,6 +5,7 @@
 		"styled-components.vscode-styled-components",
 		"stylelint.vscode-stylelint",
 		"stkb.rewrap",
-		"naumovs.color-highlight"
+		"naumovs.color-highlight",
+		"tobermory.es6-string-html"
```

Comment:
why is this added?

Reply:
this (and the associated `/* css */` comments add syntax highlighting to our css template strings — I need to update the PR description as I just added this after a conversation with @mxdvl

Reply:
Yes, I think this should live in a separate PR!

Reply:
👍  will add as a followup

Reply:
have now removed this from the PR